src/Sulu/Bundle/CoreBundle/SuluCoreBundle.php

```diff
@@ -51,7 +50,6 @@ public function build(ContainerBuilder $container)
             )
         );
 
-        $container->addCompilerPass(new ExceptionHandlerCompilerPass());
```

Comment:
the `FOS\RestBundle\Serializer\Normalizer\ExceptionHandler` was deprecated in https://github.com/FriendsOfSymfony/FOSRestBundle/pull/2161 and removed in `3.0`. we have already disabled this class via `fos_rest.exception.serialize_exceptions` in https://github.com/sulu/sulu/pull/4798

Reply:
Are you sure it works still correctly when having FosRest 2.8 installed?

Reply:
Because as I remember there the old mechanism is used where the service is used

Reply:
i have not tested it yet, but as far as i see is the `fos_rest.serializer.exception_normalizer.jms` service removed if `fos_rest.exception.serialize_exceptions` is set to `false` here:
https://github.com/FriendsOfSymfony/FOSRestBundle/blob/369149f1de7f6519960fb1bb54c73e1b8c7dddcf/DependencyInjection/FOSRestExtension.php#L431-L439

we do set this to `false` here:
https://github.com/sulu/sulu/blob/2.1/src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php#L101

Reply:
is okay for me then to remove this call